coding/mmap_reader.cpp

```diff
@@ -4,8 +4,11 @@
 
 #include <cstring>
 
-// @TODO we don't support windows at the moment
-#ifndef OMIM_OS_WINDOWS
+#ifdef OMIM_OS_WINDOWS
+  #define WIN32_LEAN_AND_MEAN
```

Comment:
ditto

Reply:
> These should be global, project-wide defines in CMakeLists.txt to avoid surprises.

I think you confused `WIN32_LEAN_AND_MEAN` with `_WIN32_WINNT`.

Reply:
No. What are the cons of defining WIN32_LEAN_AND_MEAN globally?

Reply:
Hmm none I guess, unless someone wants to include excluded stuff.

Reply:
Then let's make it global. And revise if _someone_ wants to include excluded stuff )